### ISSUE TITLE
Widget With Live Camera

### ISSUE THREAD

**Issue by ahhyeah** [IPCams/BugTracker#14]
**Is your feature request related to a problem? Please describe.**
No problem

**Describe the solution you'd like**
I would just like a widget that allows me to add live video feed from RSTP stream. Even if the FPS is low. 

**Additional context**

**Comment by fmarcoux96** [comment]
I have an idea for this: MJPEG (2nd option, so that we can still use RTSP for live).

With the new iOS 14 widgets, it would be very cool.

**Comment by tylerjonesio** [comment]
Thanks for the request! I'm looking into what can be done here. Unfortunately the OS limits how often widgets can refresh themselves on iOS 14 and the content shown can't be live interactive content (like a stream). At the moment I am looking to see what I can do with jpeg snapshots and refreshing the widget when the OS allows. It won't be as great as a live stream view, but it should be better than nothing.

**Comment by fmarcoux96** [comment]
> Thanks for the request! I'm looking into what can be done here. Unfortunately the OS limits how often widgets can refresh themselves on iOS 14 and the content shown can't be live interactive content (like a stream). At the moment I am looking to see what I can do with jpeg snapshots and refreshing the widget when the OS allows. It won't be as great as a live stream view, but it should be better than nothing.

Agreed, it would be something good already. If the app could allow multiple stream configs per camera, we could have RTSP for live and MJPEG or JPEG Snapshot (most cameras have a URL for a quick snapshot), then it would make it faster.
